content/guide/getting-started/start.html

```diff
@@ -0,0 +1,105 @@
+            <a class="hx-titlebar-menu-icon"><i class="fa fa-cog"></i><span class="hx-titlebar-menu-text">Settings</span></a>
+            <a class="hx-titlebar-menu-icon"><i class="fa fa-power-off"></i><span class="hx-titlebar-menu-text">Sign out</span></a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="hx-titlebar-linkbar">
+      <div class="hx-titlebar-contents">
+        <a class="hx-titlebar-link">Link 1</a>
+        <a class="hx-titlebar-link">Link 2</a>
+        <a class="hx-titlebar-link">Link 3</a>
+        <a class="hx-titlebar-link">Link 4</a>
+        <a class="hx-titlebar-link">Link 5</a>
+      </div>
+    </div>
+  </div>
+  <div class="hx-content">
+    <div id="card-container"></div>
+  </div>
+
+  <!-- Include the hexagon script at the end of the body -->
+  <script src="/resources/hexagon/latest/hexagon.js"></script>
+
+
+  <!-- Add your own scripts -->
+  <script>
+    function titleText (title, text) {
+      return hx.card.section()
+        .add(hx.detached('div')
+          .add(hx.card.title({text: title})))
+        .add(hx.detached('div')
+        .add(hx.card.text({text: text})))
+    }
+
+    function sparkline (options) {
+      return hx.card.group()
+        .add(hx.card.fixed.section().add(hx.card.text({text: options.text})))
+        .add(hx.card.section().add(hx.sparkline({strokeColor: options.sparklineColor, data: options.sparklineData})))
+    }
+
+    function random (i) {
+      return {
+        x: i,
+        y: Math.random()
+      }
+    }
+
+    function statusCard (name, context) {
+      return hx.card()
+        .add(hx.card.header.section({ context: context})
+          .add(hx.card.title({text: name})))
+        .add(hx.card.header.group()
+          .add(titleText('Status', 'Normal'))
+          .add(titleText('Uptime', '5 days'))
+          .add(titleText('Memory Usage', '482 MB'))
+          .add(titleText('Thread Count', '19'))
+          .add(titleText('Page Hits', '2236 / s')))
+        .add(hx.card.aligned()
+          .add(sparkline({text: 'Memory Use', sparklineColor: hx.theme.plot.colors[0], sparklineData: hx.range(60).map(random)}))
+          .add(sparkline({text: 'Thread Count', sparklineColor: hx.theme.plot.colors[1], sparklineData: hx.range(60).map(random)}))
+          .add(sparkline({text: 'Page Hits', sparklineColor: hx.theme.plot.colors[2], sparklineData: hx.range(60).map(random)}))
+          .add(sparkline({text: 'Messages Processed', sparklineColor: hx.theme.plot.colors[3], sparklineData: hx.range(60).map(random)})))
+    }
+
+    hx.select('#card-container').classed('hx-group hx-horizontal', true)
+      .add(hx.section()
+        .add(statusCard('Instance 1', undefined)))
+      .add(hx.section()
+        .add(statusCard('Instance 2', 'positive')))
+      .add(hx.section()
+        .add(statusCard('Instance 3', 'negative')))
+  </script>
+</body>
+</html>
```

Comment:
This file is used in a codeblock on the getting started page as well as in an iframe (with a link provided to open in a new page)